site/gatsby-site/src/components/cite/RemoveDuplicateModal.js

```diff
@@ -0,0 +1,194 @@
+              }
+
+              try {
+                await updateIncident({
+                  variables: {
+                    query: { incident_id: true_incident_number },
+                    set: { reports: { link: reportIds } },
+                  },
+                });
+              } catch (e) {
+                addToast({
+                  message: `Could not transfer reports to incident ${true_incident_number}.`,
+                  severity: SEVERITY.danger,
+                  error: e,
+                });
+              }
+
+              try {
+                for (const classification of classificationsData.classifications) {
+                  await updateClassification({
+                    variables: {
+                      query: { _id: classification._id },
+                      data: {
+                        ...removeTypename(classification),
+                        incidents: {
+                          link: classification.incidents
+                            .map((incident) => incident.incident_id)
+                            .filter((incident_id) => incident_id != duplicate_incident_number)
+                            .concat(true_incident_number),
+                        },
+                        reports: {
+                          link: classification.reports.map((report) => report.report_number),
+                        },
+                      },
+                    },
+                  });
+                }
+              } catch (e) {
+                addToast({
+                  message: `Could not transfer classifications to incident ${true_incident_number}.`,
+                  severity: SEVERITY.danger,
+                  error: e,
+                });
+              }
+              try {
+                for (const subscription of subscriptionsData.subscriptions) {
+                  await updateSubscription({
+                    variables: {
+                      query: { _id: subscription._id },
+                      subscription: {
+                        userId: { link: subscription.userId.userId },
+                        incident_id: { link: true_incident_number },
+                        type: subscription.type,
+                      },
+                    },
+                  });
+                }
+              } catch (e) {
+                addToast({
+                  message: `Could not transfer subscription to incident ${true_incident_number}.`,
+                  severity: SEVERITY.danger,
+                  error: e,
+                });
+              }
+
+              setSubmitting(false);
+              onClose();
+              addToast({
+                severity: SEVERITY.success,
+                message: [
+                  `Incident ${incident.incident_id} marked`,
+                  `as duplicate of ${values.duplicateIncidentId[0]}.`,
+                  `Its page will updated within 24 hours.`,
+                ].join(' '),
```

Comment:
I guess the reason for using `join` here is to avoid a long line. Personally, I prefer to have a simple string in these cases. Specially if we want to add translations with `<Trans>` or `t()`